D0_ReadPrint.ino

```diff
@@ -88,6 +96,60 @@ void WebPrint(String output)
   }
 }
 
+void PrintExecutionStarted()
```

Comment:
Please change these new functions from print functions to string builders, in keeping with the other functions in this file.